webapi/src/main/scala/org/knora/webapi/responders/v1/ListsResponderV1.scala

```diff
@@ -267,3 +293,14 @@ class ListsResponderV1(responderData: ResponderData) extends Responder(responder
     )
   }
 }
+
+object ListsResponderV1Live {
+  val layer: URLayer[TriplestoreService with MessageRelay with AppConfig, ListsResponderV1Live] = ZLayer.fromZIO {
+    for {
+      config  <- ZIO.service[AppConfig]
+      mr      <- ZIO.service[MessageRelay]
+      ts      <- ZIO.service[TriplestoreService]
```

Comment:
```suggestion
  val layer: URLayer[TriplestoreService with AppConfig, ListsResponderV1Live] = ZLayer.fromZIO {
    for {
      config  <- ZIO.service[AppConfig]
      ts      <- ZIO.service[TriplestoreService]
```

I think `MessageRelay` is not needed here.

Reply:
It is.